tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -1014,6 +1017,7 @@ public string ExpectedStatusCodesSetFromLocalAndServerOverrides(string local, st
         [TestCase("401.4,401", new string[] { "400", "401", "402", "403", "404" }, new bool[] { false, true, false, false, false })] //does not support full status codes
         [TestCase("404,401.4", new string[] { "400", "401", "402", "403", "404" }, new bool[] { false, false, false, false, true })]
         [TestCase("401.4-404.5", new string[] { "400", "401", "402", "403", "404" }, new bool[] { false, false, false, false, false })] //does not support full status codes
+        [TestCase("Foo,Bar,X-Y", new string[] { "400", "401" }, new bool[] { false, false })] //does not work with non status code values
         public void ExpectedStatusCodesParserTests(string local, string[] inputs, bool[] expected)
```

Comment:
Is it valid to test error cases where the inputs also have decimals.....   for example, inputs = { "400","401.4","402"}

Reply:
it is. There is a test case for it `[TestCase("401.4,401", new string[] { "400", "401", "402", "403", "404" }, new bool[] { false, true, false, false, false })]`

Reply:
no, the other way around.
Config = 400, 401.4, 402.1-403.2, 405
Input= 400, 401, 401.5, 402, 402.3, 404,405,405.3

Reply:
we don't support full status code as an input. I will create a false test case for this.